fuel/transformers/text.py

```diff
@@ -24,16 +24,23 @@ class NGrams(Transformer):
         this source is 'targets'.
 
     """
-    def __init__(self, ngram_order, data_stream, target_source='targets'):
+    def __init__(self, ngram_order, data_stream, target_source='targets',
```

Comment:
I'm not familiar with this transformer. I'd be more comfortable leaving this as it is and open a ticket.